content/tracing/faq/agent-5-tracing-setup.md

```diff
@@ -0,0 +1,70 @@
+---
+title: APM and Distributed Tracing with agent v5
+kind: faq
+private: true
+---
+
+## Getting started
+
+APM is available as part of the Datadog Agent with versions 5.11+ as part of the one line install for the Linux and Docker Agents. Currently, [Mac](https://github.com/DataDog/datadog-trace-agent#run-on-osx) and [Windows](https://github.com/DataDog/datadog-trace-agent#run-on-windows) users must perform a manual install of the APM Agent (aka Trace Agent) via a separate install process.
+
+The Agent can be enabled by including the following in your [Datadog agent configuration file](/agent/faq/where-is-the-configuration-file-for-the-agent):
```

Comment:
Here `agent` isn't capitalized following the word `Datadog`, but in the previous sentence it is.

> The Agent can be enabled by including the following in your [Datadog Agent configuration file](/agent/faq/where-is-the-configuration-file-for-the-agent):

Reply:
👌